packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1726,24 +1726,31 @@ Authenticators can be broadly classified into three kinds of factors. A factor r
 
 Multi-factor authentication (MFA) is the use of more than one factor. MFA is the most common way to increase assurance. Authenticators also have other characteristics that may raise or lower assurance. For example, possession factors may be implemented in software or hardware, with hardware being able to provide greater protection when storing shared secrets or private keys, and thus providing higher assurance.
 
-| Parameter            | Type              | Description                                                                                                             | Supported Values                                                                                  |
```

Comment:
Not adding a OIE only note for `Verification Method Object` and `Constraints` objects, since the parent object `#App-Sign-On-Polic`y  has it.